src/components/subscribe/subscribe.ts

```diff
@@ -23,10 +23,15 @@ export const initSubscribe = (containerEl: HTMLElement) => {
   );
   const formEl = containerEl.querySelector('form');
   const formFocusableEls = containerEl.querySelectorAll('label, input, button');
-  const controlEls = containerEl.querySelectorAll('.js-subscribe__control');
+  const controlEls = containerEl.querySelectorAll<HTMLElement>(
+    '.js-subscribe__control'
+  );
+  const controlsUiWrapper = containerEl.querySelector<HTMLElement>(
+    '.js-c-subscribe__controls-ui'
```

Comment:
Why does one of these class names include a `-c-` in it?

Reply:
Oops, I will remove the `c`. 👍🏽

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1882/commits/ca09368787159d05ef6b5f45f6890b324be8da9a